### DISCUSSION TITLE
Sorry I'm a rookie, but how do I this after I followed all the instructions, do I just go to

### DISCUSSION THREAD

**Discussion by Kforgithub** [acheong08/ChatGPT#198] [Q&A] [answered]
Sorry, I'm a rookie, but how do I this after I followed all the instructions, do I just go to the openai page and everything will be different?

**Comment by acheong08** [comment, marked as answer]
What? This does nothing to the OpenAI page. This is a library for developers to build bots. It also has a CLI/TUI for people to use in their terminal.

**Reply by Kforgithub** [reply]
Oops, got it. Sorry for asking such a rookie question.😂 Cheers.
